### PR TITLE
TBS - Updates broken/incorrect links in MyAccount API docs

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Updates broken links in MyAccount API docs.
- **Is this PR related to a Monolith release?** No.

### Resolves:

* N/A

### CODE REVIEW

**Review by hanyuanzhang-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/guides/configure-user-scoped-account-management/main/index.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This guide explains how to set up end-user driven account management in your org
 
 [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/9cb68745dbf85ae3a871)
 
-> **Note:** To run the Postman collection, you need an end-user access token. Use an [SDK](/docs/guides/auth-js/-/main/) to get the token.
+> **Note:** To run the Postman collection, you need an end-user access token. Use an [SDK](/docs/guides/auth-js/-/main/#handle-responses) to get the token.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-user-scoped-account-management/main/index.md:37]:
```suggestion
> **Note:** To run the Postman collection, you need an end-user access token. Use an [SDK](/docs/guides/auth-js/main/#handle-responses) to get the token.
```

File: packages/@okta/vuepress-site/docs/reference/api/myaccount-migration/index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ The `idp` version of the MyAccount API only works on Okta Identity Engine. See [
 
 ## Use the Postman collection
 
-To run the Postman collection of the `idp` version of the API, you need an end-user access token. Use an [SDK](/docs/guides/auth-js/-/main/) to get the token.
+To run the Postman collection of the `idp` version of the API, you need an end-user access token. Use an [SDK](/docs/guides/auth-js/-/main/#handle-responses) to get the token.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/myaccount-migration/index.md:18]:
```suggestion
To run the Postman collection of the `idp` version of the API, you need an end-user access token. Use an [SDK](/docs/guides/auth-js/main/#handle-responses) to get the token.
```

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ The Okta MyAccount API allows end users to fetch and update their own Okta user
 
 * This API is only for Okta Identity Engine. If you’re using Okta Classic Engine, see [MyAccount API (deprecated)](/docs/reference/api/archive-myaccount/). See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
 * Explore the MyAccount API:
-   > **Note:** To run this Postman collection, you need an end-user access token. Use a password grant type or SDK to get the token. See [Embedded Auth with SDKs React](https://github.com/okta/okta-auth-js/tree/master/test/apps/react-oie) for a sample app that provides an access token.
+   > **Note:** To run the Postman collection, you need an end-user access token. Use an [SDK](/docs/guides/auth-js/-/main/#handle-responses) to get the token.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:23]:
```suggestion
   > **Note:** To run the Postman collection, you need an end-user access token. Use an [SDK](/docs/guides/auth-js/main/#handle-responses) to get the token.
```

**Review by vanngo-okta — COMMENTED** [review]
You don't need the `/-/` since there's no stack selector folders implemented yet.
Also, if you add the `/-/`, it breaks the left-nav and breadcrumbs (an existing bug :-( )

**Review by vanngo-okta — APPROVED** [review]
LGTM 🚀

**Pull request merged**